.github/workflows/test.yaml

```diff
@@ -82,6 +82,26 @@ jobs:
         with:
           aqua_version: v1.25.0
 
+      - name: Generate image list digest
+        id: image-digest
+        run: |
+          IMAGE_LIST=$(skopeo list-tags docker://ghcr.io/aquasecurity/trivy-test-images)
+          DIGEST=$(echo "$IMAGE_LIST" | sha256sum | cut -d' ' -f1)
+          echo "digest=$DIGEST" >> $GITHUB_OUTPUT
+
+      - name: Restore test images from cache
+        uses: actions/cache@v4
+        id: restore-test-images
+        with:
+          path: integration/testdata/fixtures/images
+          key: cache-test-images-${{ steps.image-digest.outputs.digest }}
+          restore-keys:
+            cache-test-images-
+
+      - name: Download test images
+        if: steps.restore-test-images.outputs.cache-hit != 'true'
+        run: mage test:fixtureContainerImages
+
```

Comment:
We don't need this step anymore. It is transparently called.
https://github.com/aquasecurity/trivy/blob/bbc8e1d8f3e27726ca3d413f7481e7f62814e7b8/magefiles/magefile.go#L277

And it's efficient as it skips images if it exists locally.
https://github.com/aquasecurity/trivy/blob/bbc8e1d8f3e27726ca3d413f7481e7f62814e7b8/magefiles/fixture.go#L30-L32

Now, we call this check twice, which is just time-consuming (and API-consuming).

Reply:
Right, since the cache will still be saved at the end of the job!